layers/+source-control/git/README.org

```diff
@@ -59,6 +64,24 @@ file.
 Of course if your OS does not ship with git (!) you'll have to install it
 on your machine. You can download it from the [[http://git-scm.com/downloads][download page]].
 
+*** Basic Git configuration
+Define your git identity using the following commands in a terminal window
+replacing username and name@domain.tld with your own values
+
+#+begin_src shell
+  git config --global user.name "username"
+  git config --global user.email "name@domain.tld"
+#+end_src
+
+*** Using SSH URLs for remote repositories
+When using an SSH URL for remote repository access, [[https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/][generate an SSH key]] and
+add that key to your account on the remote service that manages repository access.
+An SSH key removes the need to provide login details for each request from Magit
+to the remote repository service.
+
+- [[https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account][GitHub /documentation]]
```

Comment:
Is the `/` before "documentation" on purpose?

Reply:
fixed